src/test/components/__snapshots__/TimelineMarkers.test.js.snap

```diff
@@ -5,7 +5,7 @@ exports[`renders TimelineMarkers correctly 1`] = `
   className="timelineMarkers"
```

Comment:
Might be easier to review this by looking at individual commits, as I have a first commit to update the snapshots on master code using the new test case, and then a second commit to show the snapshot changes induced by the code changes.